.gitignore

```diff
@@ -19,3 +19,7 @@ __TODO.txt
 
 # tmp images
 *.jpg
+/.idea/
+/.idea/
+.idea/*
+/.idea/*
```

Comment:
I thought `.idea/*` would work, but apparently not -- there are still `.idea` files

Try `/**/.idea/*`. If it doesn't work, try the suggestions here -- https://stackoverflow.com/questions/11968531/what-to-gitignore-from-the-idea-folder